pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1050,19 +1050,20 @@ func (self *awsInstance) getMountDevice(volumeID string, assign bool) (assigned
 		return mountDevice(""), false, nil
 	}
 
-	// Check all the valid mountpoints to see if any of them are free
-	valid := instanceType.getEBSMountDevices()
```

Comment:
Is there another caller of `getEBSMountDevices()`  or should you be deleting that code as well?